sdk/analysisservices/arm-analysisservices/vitest.browser.config.ts

```diff
@@ -1,17 +0,0 @@
-
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-import { defineConfig, mergeConfig } from "vitest/config";
-import viteConfig from "../../../vitest.browser.shared.config.ts";
-
-export default mergeConfig(
```

Comment:
curious why this file is removed?

Reply:
Because we are currently not running the unit tests for the browser, so no there is no need in keeping them.

Reply:
Restoring files back